content/en/blog/test-coverage/index.md

```diff
@@ -0,0 +1,141 @@
+---
+date: 2023-11-16
+title: "Test Coverage Beyond Unit Testing"
+linkTitle: "Coverage for E2E Tests"
+description: "Automated testing is like having an extra member in your team, a knight against regression. Test coverage measurements and automatic test coverage monitoring help the team keep the knight shiny, i.e., automated test sets in good shape. Traditionally, we have measured the coverage for unit tests, but thorough CI pipelines also include other types of testing. Go has recently introduced new tooling that allows us to measure the test coverage for application tests and thus improve our capabilities for keeping our automated tests in shape automatically."
+author: Laura Vuorenoja
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+
+Automated testing is your superpower against regression. It prevents you from breaking existing
+functionality when introducing new code changes. The most important aspect of automated testing
+is **automation**. The testing should happen automatically in the continuous integration (CI) pipeline
+whenever developers create pull requests. If the tests are not passing, the team doesn't merge
```

Comment:
Why only for PRs? 

In my repos, I have, quite clumsily, still in GitHub actions, but somehow, I have different types of test automation pipelines, i.e., some of them are triggered just with the commit&push.

Is this a security issue or resource waste? But I don't get it why to limit PRs?